config/registration_numbers.yml

```diff
@@ -1,95 +1,4 @@
-# registration numbers for verification document МИ 1592-2015
-- 19653-00
-- 19653-05
-- 19653-07
-- 58329-14
-- 60780-15
-- 49152-12
-- 43682-10
-- 33952-07
-- 29146-05
-- 13671-06
-- 48241-11
-- 16078-00
-- 16078-05
-- 16078-13
-- 61648-15
-- 44572-10
-- 54835-13
-- 61399-15
-- 66411-17
-- 75657-19
-- 17041-03
-- 24860-03
-- 24860-11
-- 24860-16
-- 27361-04
-- 27361-09
-- 49316-12
-- 57034-14
-- 33541-06
-- 33541-12
-- 51794-12
-- 47153-11
-- 55359-13
-- 47244-11
-- 57032-14
-- 55467-13
-- 60909-15
-- 67831-17
-- 54233-13
-- 23556-02
-- 29438-05
-- 24319-03
-- 24319-05
-- 38760-08
-- 48411-11
-- 58361-14
-- 56279-14
-- 25986-03
-- 25986-04
-- 32819-06
-- 37584-08
-- 54418-13
-- 32919-06
-- 42813-09
-- 42813-14
-- 56505-14
-- 26382-04
-- 26382-07
-- 26382-12
-- 58362-14
-- 13869-03
-- 13869-08
-- 13869-13
-- 31697-06
-- 39392-08
-- 60378-15
-- 27014-04
-- 53352-13
-- 69675-17
-- 27267-04
-- 27270-04
-- 59828-15
-- 17104-05
-- 20832-05
-- 31107-06
-- 32538-06
-- 32538-11
-- 32539-06
-- 32539-11
-- 61032-15
-- 66635-17
-- 31125-06
-- 51333-12
-- 46597-11
-- 63458-16
-- 13544-93
-- 13667-01
-- 13668-06
-- 13669-06
-- 13670-06
-- 13672-06
-- 13673-06
-- 75657-19
-- 68501-17
\ No newline at end of file
+method1: # add one of the methods you work with
```

Comment:
It's better to make this a *real looking* example with some data to and name it something like `config/registration_numbers.yml.dist` while also tracking it with Git and not tracking the final `config/registration_numbers.yml` config.